packages/common-tests/browser-tests/page-model/landingPage.ts

```diff
@@ -28,8 +28,9 @@ class LandingPage {
       screen.getByPlaceholderText(searchPlaceholderText),
       this.searchText
     );
-    await t.wait(2000);
-    await t.expect(screen.findByText(this.searchText).exists).ok();
+    await t
+      .expect(screen.findByText(this.searchText).exists)
+      .ok({ timeout: 5000 });
```

Comment:
I think that the adding general PageRequestTimeout from testcafe settings would be most robust way to fix this:
https://testcafe.io/documentation/402638/reference/configuration-file#pageRequestTimeout or 
https://testcafe.io/documentation/402638/reference/configuration-file#pageloadtimeout 

Hiccups are usually more related to the whole environment rather that single page,  so timeout settings could be set straight to the `yarn browser-test` CLI command because then `yarn browser-test:ci` command could have different (bigger) timeout values platta might have even more hiccups than local machine.

Reply:
this might be a wrong solution. I also tried `t.wait(5000)`, but the result was the same. This timeout is here for the debounce-feature and it just seems to take some time that the autocomplete answers appear.

Reply:
I don't think that using a global page request timeout is a solution here either. I think I'll just add some comments next to the timeout so it's more understandable.

Reply:
This is where it always fails 
![image](https://user-images.githubusercontent.com/389204/218418998-16d16c20-7a2f-4385-a6d8-84939fc3a29e.png)

This is what we should have
![image](https://user-images.githubusercontent.com/389204/218419299-012397ff-9eed-4634-b41f-9c159ac86311.png)

So, it seems like it has missed with the cookie consent modal, because it is minimized, not closed, but it also has missed to hit the search input (the autocomplete field).

Reply:
After using a t.wait() instead the timeout for ok, I also got this, where the menu has not yet appeared. 
<img width="1146" alt="image" src="https://user-images.githubusercontent.com/389204/218420824-bb36f2a9-cf66-417f-9d03-96cc877fd71f.png">

Reply:
the situation starts from this: 
![image](https://user-images.githubusercontent.com/389204/218445510-9f3d702d-82a8-4283-9f92-532aac5867a3.png)

The cookie consent modal is still open and the testcafe clicks the palceholder, so the modal gets minified.

Reply:
I refactored the initial "show cookie consent modal" state set. Now the modal should appear only if the cookie is not yet filled.